doc/development.md

```diff
@@ -104,3 +104,32 @@ The test should be added to the init function inside of the test file. If the
 test module is being created then an `init` function should be created which
 registers the tests and the package must be imported inside of
 `tests/registry/registry.go` to allow for discovery.
+
+## Marking an experimental spec as stable
```

Comment:
Needs a note about writing docs about new features.
Needs a note about copying the current `config/types` directory to `config/vX_Y/types`

Reply:
Copying `config/types` to `config/vX_Y/types` will not be true after https://github.com/coreos/bugs/issues/2346 happens, so I'm going to skip that. I more intended this to be a list of the end side-effects instead of implementation instructions anyway. Implementation instructions will be added after that refactor.